pkg/api/types.go

```diff
@@ -1146,10 +1146,97 @@ const (
 )
 
 // Affinity is a group of affinity scheduling rules, currently
-// only node affinity, but in the future also inter-pod affinity.
+// includes node affinity,pod affinity & pod anti affinity.
 type Affinity struct {
 	// Describes node affinity scheduling rules for the pod.
 	NodeAffinity *NodeAffinity `json:"nodeAffinity,omitempty"`
+	// Describes pod affinity scheduling rules for the pod.
```

Comment:
Unlike "node affinity" this might be unfamiliar enough to people that we should explain a little more. Maybe "Describes pod affinity scheduling rules (e.g. place this pod on nodes that already have certain other pod(s))."